src/main/scala/net/snowflake/spark/snowflake/Utils.scala

```diff
@@ -643,4 +645,10 @@ object Utils {
         println(s"Fail to print result set: ${e.getMessage}")
     }
   }
+
+  /**
+   * Generate random suffixes; useful for use
+   * in generating collision-safe identifiers in tests.
+   */
+  def randomSuffix: String = Math.abs(Random.nextLong()).toString
```

Comment:
I think randomSuffix is for testing purpose. if so, can you add it to test TestUtils.

Reply:
Moved.